Formula/streem.rb

```diff
@@ -0,0 +1,17 @@
+class Streem < Formula
+  desc "prototype of stream based programming language"
+  homepage "https://github.com/matz/streem"
+  url "https://github.com/matz/streem/archive/201611.tar.gz"
+  sha256 "807a9ccf3162efb061a15a333943050b329f0232cceb55492e5cd582c09a7fb4"
+  head "https://github.com/matz/streem.git"
+
+  def install
+    system "make"
+    cp_r "bin", prefix
```

Comment:
I will ask the developers, but can not you just merge with this? Probably I will make it take time, including pull requests.